app/views/idv/otp_verification/show.html.erb

```diff
@@ -0,0 +1,38 @@
+<% title t('titles.enter_2fa_code') %>
+
+<h1 class="h3 my0">
+  <%= t('two_factor_authentication.header_text') %>
+</h1>
+
+<p>
+  <%= @presenter.phone_number_message %>
```

Comment:
Given the source...

```slim
p == @presenter.phone_number_message
```

Should this be...

```suggestion
  <%== @presenter.phone_number_message %>
```

?

**Edit:** Just tested, seems fine as-is. I'm assuming it's because `phone_number_message` using `t` with `_html`-suffixed key will mark as HTML safe.

Reply:
yep, if it has the `_html` it should be good